Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2837,8 +2797,6 @@ - (void)_didStopDeferringGeometryUpdates
     if (!_perProcessState.lastSentDeviceOrientation || newOrientation != _perProcessState.lastSentDeviceOrientation.value())
         [self _dispatchSetDeviceOrientation:newOrientation];
 
-    [self _updateDrawingAreaSize];
```

Comment:
(specifically I mean here too)

Reply:
I'll try this out - but I wonder if this is why the issue reproduces after your change? Because the current behavior relies on `if (![_contentView sizeChangedSinceLastVisibleContentRectUpdate])` on line 2568 of WKWebViewIOS.mm

Reply:
It seems fine :) I added it back